ksql-functional-tests/src/test/resources/query-validation-tests/magic-timestamp-conversion.json

```diff
@@ -0,0 +1,178 @@
+{
```

Comment:
should we ad RQTT tests to correspond for pull queries?

Reply:
There are tests already